scrapenews/spiders/timeslive.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+from .sitemap import SitemapSpider
+from scrapenews.items import ScrapenewsItem
+from datetime import datetime
+import pytz
+
+SAST = pytz.timezone('Africa/Johannesburg')
+
+
+class TimesliveSpider(SitemapSpider):
+    name = 'timeslive'
+    allowed_domains = ['www.timeslive.co.za']
+
+    sitemap_urls = ['https://www.timeslive.co.za/robots.txt']
+    sitemap_follow = [
+        'www.timeslive.co.za/anc-conference-2017',
```

Comment:
watch out - I think these sitemap regexes should be `www.timeslive.co.za/sitemap/...`